provisioning/service/src/Microsoft.Azure.Devices.Provisioning.Service.csproj

```
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.18.1</Version>
+    <Version>2.0.0-preview</Version>
```

```suggestion
    <Version>2.0.0-preview001</Version>
```

To follow the pattern elsewhere?